src/helpers/daily-data-providers/combined-sleep-v2.ts

```diff
@@ -18,6 +19,10 @@ export default async function (startDate: Date, endDate: Date): Promise<{ [key:
     if (settings.queryableDeviceDataTypes.find(s => s.namespace == "AppleHealth" && s.type == "SleepAnalysisInterval")) {
         providers.push(appleHealthSleepMinutesDataProvider(startDate, endDate));
     }
+    if (settings.healthConnectEnabled) {
```

Comment:
Should query MyDataHelps.getDeviceDataV2AllDataTypes() and check to make sure that sleep is enabled for Health Connect before including it here